app/styles/app.scss

```diff
@@ -1385,45 +1384,94 @@ section {
       //margin-top: 40px;
 
       .external-video {
-        height: calc(100vh)!important;
+        height: calc(100vh - 140px) !important;
       }
     }
 
-    .section-heading {
-      position: absolute;
-      padding: 10px;
-      height: 65px;
-      color: white;
-      width: 100%;
-      font-size: 38px;
-      background: #363636;
-      display: flex;
-      padding-left: 10px;
+    &:hover {
+      .section-heading {
+        display: flex;
+      }
+    }
+  }
 
-      p {
-        min-width: 95%;
-        font-size: 20px;
-        margin: 14px 0;
+  .section-heading {
```

Comment:
Please check this.

You don't need to use 
```
position: fixed;
```
for section heading I guess,
Better way is to use position absolute

Also, I am not sure why this class has been over-written, this can affect all heading for video, external video.
Please don't do this. 😟